### ISSUE TITLE
[Feature Request] Choose local names or english names for each country individually

### ISSUE THREAD

**Issue by bambamal** [osmandapp/Osmand#4921]
For obvious reasons i want all names in my homecounry (or countries that im able to speak the language of) displayed in the local language. But i want other countries to be displayed in english.

I can't just display all names in my language (german), because the osm data is so incomplete, its useless, only big city names, but no street names for example. And for international travelling you need the english names anyway.

One use case is international travel planning. I constantly have to switch between english and local.

**Comment by Mannshoch** [comment]
A kind of Priority setting would be nice.
1.) Application-Language e.g. German
2.) English (maybe marked with a flag 🇬🇧 )
3.) Local

Maybe an idea.
If Osmand detect a localisation in a country which has not an Official language like set-up as Application-Language maybe a button on screen should appear to switch the Language of the names instant.

**Comment by bambamal** [comment]
Good idea. It just it wouldnt help users speaking multiple languages/living in different countries. Also it might result in some countries being partly partly german and partly english.

Ideally you could setup something like this:
1) Germany, Austria, Switzerland in German
2) France in French
3) Everything else in English
4) Local only if there is no other option

The easiest way to set this up in GUI would IMO be a setting:
1) Use following language (e.g. english) (first menu)
2) Except use local in following countries (second menu)

**Comment by johannes-mueller** [comment]
I would vote for always local except if the local language uses a non latin alphabet. The whole concept of translating geographical names is just weird to me.

**Comment by beatnickgr** [comment]
@bambamal this makes no sense :
If an italian speaker goes to egypt and does not speak english, why would he put his settings in english?
osmand~ should detect that the country being viewed is not in the users locale and automatically use int_name.
I totally agree that it makes no sense to translate each street in the 200 available languages of osmand~ especially if those translations are not official and therefore the user should guess what orthography the cartographer has used.
